scrapy/utils/project.py

```diff
@@ -12,8 +12,8 @@
 
 
 def inside_project():
-    scrapy_module = os.environ.get("SCRAPY_SETTINGS_MODULE")
-    if scrapy_module is not None:
+    scrapy_module = os.environ.get(ENVVAR)
+    if scrapy_module:
```

Comment:
This is a change in behavior when the `SCRAPY_SETTINGS_MODULE` is set to an empty string. That said, an empty string cannot work here, so I guess it is fine and slightly improves readability, so +1 from me.

Reply:
Yeaaaaah!!!! =]